corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -373,6 +373,12 @@ hqDefine("cloudcare/js/formplayer/app", function () {
                 }
             }
         );
+
+        window.addEventListener(
```

Comment:
Should there be a corresponding `afterprint` function that removes the class? Seems like there's an edge case where someone prints multiple times and interacts with the form between print actions.

Reply:
Ah, good catch. Is this a clean way of doing it?

        let lastPanelElement;
        
        window.addEventListener(
            'beforeprint', function () {
                lastPanelElement = $('.panel.panel-default, .q.form-group').last();
                if (lastPanelElement) {
                    lastPanelElement.addClass('last');
                }
        });
        
        window.addEventListener(
            'afterprint', function () {
                lastPanelElement.removeClass('last');
        });

Reply:
That'll do it correctly, yes, although I'd be inclined to use `$(".last").removeClass("last")` rather than increasing the [span / live time](http://www.rosscode.com/blog/index.php?title=variable_usage_span_and_live_time&more=1&c=1&tb=1&pb=1) (I always forget which is which) of `lastPanelElement`.